CMakeLists.txt

```diff
@@ -125,15 +127,13 @@ elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
     set(CMAKE_PREFIX_PATH
         ${CMAKE_PREFIX_PATH}
        "${CMAKE_SOURCE_DIR}/external/precompiled/linux/lib/arm64-v8a")
-elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "arm")
-    set(CMAKE_PREFIX_PATH
-        ${CMAKE_PREFIX_PATH}
-       "${CMAKE_SOURCE_DIR}/external/precompiled/linux/lib/arm")
 endif()
 
 # GFXReconstruct provided find modules
 find_package(LZ4)
-find_package(ZSTD)
+if(BUILD_ZSTD_SUPPORT)
```

Comment:
I sent this as an email to you last week, but maybe it went into spam: The consensus internally is that if you pass ZSTD_ROOT for CMake on the command line to the root dir of a ZSTD download, FindZSTD.cmake should find your include and lib and use those in the build.  That's actually what our internal CI does.  So I don't think you need to specify anything related to ZSTD in this PR.